samcli/commands/exceptions.py

```diff
@@ -19,6 +19,11 @@ class UserException(click.ClickException):
 
     exit_code = 1
 
+    def __init__(self, message, wrapped_from=None):
+        self.wrapped_from = wrapped_from
```

Comment:
I like the idea. Does this information need to be surfaced somewhere though?

Reply:
In telemetry (looks like you found it, base on your next comment :))